OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+
+## 5. Building Oracle WebCenter Sites Docker Images
+1. Go to the folder where Docker scripts are located:
+```
+    $ cd ../docker-images-master/OracleWebCenterSites/dockerfiles
+```
+
+2. To build Oracle WCS Docker image, run the following command:
+```
+   $ sh buildDockerImage.sh -v 12.2.1.3
+```
+   For more information on usage of <i>buildDockerImage.sh</i> command, see [FAQ](#1-what-is-the-usage-of-builddockerimagesh-file),
+
+**IMPORTANT**: The resulting image has the automated scripts to:
+-  Run RCU
+-  Create and configure a WebLogic domain
+-  Run WebCenter Sites Configuration process while creating an admin container out of the image
+
+## 6. Preparing to Run Oracle WebCenter Sites Docker Image
+Before running the Oracle WebCenter Sites Docker image, you need to set up a communication network between containers on the same Docker engine’s host. You also need to set up data volume to enable file sharing between Docker containers and database container.
+
+Configure an environment to run the WebCenter Sites Docker image by following these steps:
+
+##### A. Creating a user-defined network
+##### B. Mounting host directory as a data volume 
+##### C. Setting up an Oracle Database Docker container
+
+### A. Creating a user-defined network
+Create a user-defined network to enable containers to communicate by running the following command:
+```
+   $ docker network create -d bridge <network_name>
+```
+Sample command:
+```
+   $ docker network create -d bridge WCSitesNet
+```
+### B. Mounting host directory as a data volume
+You need to mount volumes, which are directories stored outside a container's file system, to store database data files and WLS domain files. The default location of the volume in the container is "/var/lib/docker/volumes". 
+
+This option lets you mount a directory from your Docker engine’s host to a container as volume. This volume is used to store database data files and WebLogic server domain files. The volume is created at this location <i>/scratch/DockerVolume/WCSitesVolume/</i>.
+
+To mount a host directory as a data volume, execute the below commands as **root** user.
+```
+   $ mkdir -p /scratch/DockerVolume/WCSitesVolume/WCSites /scratch/DockerVolume/WCSitesVolume/WCSitesShared
+   $ chmod -R 777 /scratch/DockerVolume/WCSitesVolume
+```
+All container operations are performed as 'oracle' user. Therefore, the ‘chmod’ command must be run to grant permission to all the non-root users.
+
+### C. Setting up an Oracle Database Docker container
+To set up an Oracle Database Docker container, you must first update the environment file which is passed as a parameter in the command that starts the database container.  
+
+##### 1. Update the Environment File
+Update the environment **db.env.list** file, to define the parameters, which is located at `../docker-images-master/OracleWebCenterSites/dockerfiles/db.env.list`.
```

Comment:
Please use backticks instead of making the filename bold.

Reply:
Changes incorporated into latest file.